modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/servlet/taglib/BaseContainerTag.java

```diff
@@ -362,9 +366,9 @@ protected int processStartTag() throws Exception {
 		jspWriter.write(processCssClasses(new LinkedHashSet<>()));
 		jspWriter.write("\"");
 
-		if (Validator.isNotNull(_id)) {
+		if (Validator.isNotNull(getId())) {
 			jspWriter.write(" id=\"");
-			jspWriter.write(_id);
+			jspWriter.write(getId());
 			jspWriter.write("\"");
```

Comment:
@carloslancha do I need to apply these changes in my "future" PR? 
Just asking.gif

Reply:
@julien I put all my `BaseContainerTag` and `BaseClayCard` changes in [a separate commit](https://github.com/liferay-frontend/liferay-portal/pull/449/commits/2b7fb0b158509fa5a3bdd996563c1d35a5939658). That's the code we all need. If we are pessimistic about this PR being merged quickly, it would be a good idea that @carloslancha makes such a commit, double-checking if anything is missing. Then, two of us can cherry-pick it to keep it synced.

Reply:
Since we are focusing on merging this one first as we talked in Slack, you'll just need to rebase with master then.

Reply:
@carloslancha agreed